src/rockstor/system/tests/test_services.py

```diff
@@ -68,6 +68,9 @@ def test_init_service_op_valid(self):
             "nut-server",
             "rockstor-bootstrap",
             "rockstor",
+            "replication",
```

Comment:
```suggestion
            "rockstor-replication",
```
@Hooverdan96 Looks like I left this dangling as part of my last wave of suggestions!